meteor-app/imports/startup/server/parts-data.js

```diff
@@ -0,0 +1,41 @@
+import { Meteor } from 'meteor/meteor' // base
+import Parts from '/imports/api/parts/parts'
+import casual from 'casual' 
+
+
+Meteor.methods({
+'seed.parts'() {
+  const n = 10
+// seed ensures same data is generated
+  casual.seed(123)
+
+  const arrayOf = function (times, generator) {
+    let result = [];
+    for (let i = 0; i < times; ++i) {
+      result.push(generator());
+    }
+    return result;
+  };
+
+  casual.define('part', function () {
+    return {
+      imageUrl: '/public/images/logo-large.jpg',
+      wholesalePrice: casual.integer(0, 20),
+      retailPrice: casual.integer(0, 20),
+      number: casual.integer(600000, 1000000),
+      desc: casual.short_description,
+      barcode: casual.integer(600000, 1000000),
+    }
+  })
+
+  const partArray =
```

Comment:
I think the parts list will be an object

Reply:
Can we confirm you want Parts to be an Object rather than Array ? I'm just not sure if that what you mean, if so I see good reason for this, you could potentially find parts faster in an object if there are 17 000 items rather than in Array,

Reply:
It's an array of objects